stdlib/Markdown/docs/src/index.md

```diff
@@ -1 +1,381 @@
 # Markdown
+
+This section describes Julia's markdown syntax, which is enabled by the
+Markdown standard library. The following Markdown elements are supported:
+
+## Inline elements
+
+Here "inline" refers to elements that can be found within blocks of text, i.e. paragraphs. These
+include the following elements.
+
+### Bold
+
+Surround words with two asterisks, `**`, to display the enclosed text in boldface.
+
+```
+A paragraph containing a **bold** word.
+```
+
+### Italics
+
+Surround words with one asterisk, `*`, to display the enclosed text in italics.
+
+```
+A paragraph containing an *emphasized* word.
+```
+
+### Literals
+
+Surround text that should be displayed exactly as written with single backticks, ``` ` ``` .
+
+```
+A paragraph containing a `literal` word.
+```
+
+Literals should be used when writing text that refers to names of variables, functions, or other
+parts of a Julia program.
+
+!!! tip
+    To include a backtick character within literal text use three backticks rather than one to enclose
+    the text.
+
+    ```
+    A paragraph containing a ``` `backtick` character ```.
+    ```
+
+    By extension any odd number of backticks may be used to enclose a lesser number of backticks.
+
+### ``\LaTeX``
+
+Surround text that should be displayed as mathematics using ``\LaTeX`` syntax with double backticks,
+``` `` ``` .
+
+```
+A paragraph containing some ``\LaTeX`` markup.
+```
+
+!!! tip
+    As with literals in the previous section, if literal backticks need to be written within double
+    backticks use an even number greater than two. Note that if a single literal backtick needs to
+    be included within ``\LaTeX`` markup then two enclosing backticks is sufficient.
+
+!!! note
+    The `\` character should be escaped appropriately if the text is embedded in a Julia source code,
+    for example, ``` "``\\LaTeX`` syntax in a docstring." ```, since it is interpreted as a string
+    literal.
```

Comment:
Should `raw""` be mentioned here too?

Reply:
This is just copy pasted from the other file, but I will add that.